compiler/rustc_resolve/src/lib.rs

```diff
@@ -1137,7 +1136,7 @@ impl DefIdTree for ResolverTree<'_> {
 impl<'a, 'b, 'tcx> DefIdTree for &'a Resolver<'b, 'tcx> {
     #[inline]
     fn opt_parent(self, id: DefId) -> Option<DefId> {
-        ResolverTree(&self.untracked).opt_parent(id)
+        ResolverTree(&self.tcx.untracked()).opt_parent(id)
```

Comment:
Can this use the `impl DefIdTree for TyCtxt` directly? If so, can we remove `ResolverTree` helper completely?

Reply:
Still need to do that, may do it after landing this PR or on monday